tests/integration/julia/sandbox-mj.html

```diff
@@ -519,7 +531,7 @@
             <span data-f-bind="salesSq[4]"></span>.
         </td>
         <td class="pass">
-            pass (0.9.0). note that this only works for span, not for input (i'm assuming that's by design).
+            pass (0.10.0). note that this only works for span, not for input (i'm assuming that's by design).
```

Comment:
I don't see anything here which shouldn't work for inputs - which part isn't working?